app/serializers/api/variant_serializer.rb

```diff
@@ -41,10 +41,10 @@ def thumb_url
   end
 
   def unit_price_price
-    (rand * 10).round(2)
+    price_with_fees / VariantUnits::UnitPrices.new(object).unit_price_denominator
   end
 
   def unit_price_unit
-    rand > 0.5 ? "item" : "kg"
+    VariantUnits::UnitPrices.new(object).unit_price_unit
   end
 end
```

Comment:
I'm wondering if there is any opportunities to create a structure for `unit price`, something like : 
`unit_price = {
price: # the price
unit: # the unit
}`
And then modify the FE, but should be very slightly changes. I could handle it.

Reply:
I think I'd keep it split up like this... it keeps the API responses simpler if we're only returning strings and not hashes that could behave in slightly different ways depending on the context. Plus it's already working the way it is, so unless there's a specific reason to change it, I think we should keep it.